ui/plugins/eu.esdihumboldt.hale.ui.functions.groovy/src/eu/esdihumboldt/hale/ui/functions/groovy/internal/TypeStructureTray.java

```diff
@@ -393,14 +396,15 @@ private String createSourceSample(TreePath path, Collection<? extends TypeDefini
 			}
 			else {
 				// assuming Retype/Merge
```

Comment:
_Explanation:_
This comment is related to that previously if was identified that this (a) is not a join and (b) this is not a property transformation.
It can't be a Groovy Create either because that has no source.